internal/compiler/widgets/cosmic-base/spinbox.slint

```diff
@@ -54,6 +54,7 @@ export component SpinBox {
     accessible-value-minimum: root.minimum;
     accessible-value-maximum: root.maximum;
     accessible-value-step: (root.maximum - root.minimum) / 100;
+    forward-focus: base;
```

Comment:
why are all these forward-focus there in the widgets?
was there a behavior change?

Reply:
Since "forward-focus" was a reserved property, an omission was not detected by the style checker. But if you write something like this:

```
import { SpinBox } from "std-widgets.slint";

export component Test {
    spin := SpinBox {}
    TouchArea {
        clicked => { spin.focus(); }
    }
}
```

then it will work with the material style, but it will fail to compile with the other styles.

Reply:
Aha, good point. We should update the style checker to make sure this is covered.
But this is orthogonal for this PR